dev-python/prov/prov-1.5.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="W3C provenance data dodel library"
+HOMEPAGE="https://pypi.python.org/pypi/prov"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="
+	dev-python/pydotplus[${PYTHON_USEDEP}]
```

Comment:
In the light of #5478, I'm guessing we need to change our depend to `dev-python/pydot` and it should all work right?